### ISSUE TITLE
Accessibility finding: "We can't match your records" page doesn't have an H1

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#139066]
### Checklist item

WEB-131-003 There is one H1 per page/screen

### Severity level

2, High. Should be fixed in 1-2 sprints post-launch.

### Page(s)/url(s)

https://staging.va.gov/my-va/submissions/9093f30e-0d09-43ff-a13f-087847063071

### Details

"We can't match your records" page: The only heading on this page is an H2, "We can't match your records."

### Reproduction steps

Log in as user 189, then go to the page. Inspect the heading.

### Proposed solution or next steps

Make "We can't match your records" an H1 instead of an H2.
